### DISCUSSION TITLE
TS011F_din_smart_relay change indicator Led color

### DISCUSSION THREAD

**Discussion by erwan33fr** [Koenkk/zigbee2mqtt#12244] [Devices]
https://www.zigbee2mqtt.io/devices/TS011F_din_smart_relay.html

By default, on TS011F_din_smart_relay, the indicator LED is green in the off state and red in the on state.
(which doesn't really make sense)

According to the seller, it is possible to reverse the color of the indicator  LED by configuration.

How to modify this parameter in zigbee2mqtt?

**Comment by Matriciel** [comment]
Hi, in France, all din systems are like this: RED is on, GREEN is off

**Comment by erwan33fr** [comment]
![Capture](https://user-images.githubusercontent.com/86933399/164774209-3d8d6b99-7f7f-4497-9541-b962f296d8fc.JPG)

On the Tuya application, there are many specific options concerning the LED (such as the one that allows you to choose the state on restart).

What bothers me is mainly to have a green LED on permanently.
The red LED when the status is ON does not bother me.
Via the Tuya app it is possible to configure this.
This must also be possible via Zigbee2mqtt by updating the code?
